proposals/Exceptions.md

```diff
@@ -1,302 +1,17 @@
 # Exception handling
 
 So far there have been two proposals ([first proposal](old/Exceptions.md),
-[second proposal](old/Level-1.md)) proposed. We are going to use the second
-proposal for the binary section spec, but we have not decided on which one to
-use for the instruction part of the spec. This document describes status quo of
-the exception handling proposal: for some parts we present both candidate
-options until we fully decide.
+[second proposal](old/Level-1.md)) proposed. In the TPAC meeting in Oct 2018, we
+[took a vote and decided to use the second
+proposal](https://github.com/WebAssembly/meetings/blob/master/2018/TPAC.md#exception-handling-ben-titzer)
+with the first-class exception type mainly based on the reasoning that it is
+more expressive and thus also more extendible to other kinds of events. This
```

Comment:
I'd remove "thus" since the expressiveness is a feature even independently of extensibility.

Reply:
Done